app/build.gradle

```
@@ -38,6 +38,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
+    implementation 'com.google.firebase:firebase-firestore:21.7.1'
```

Because we have `com.google.firebase:firebase-bom:25.12.0` we don't need to specify version. Following Google's best practices, it's best to use the following instead
```suggestion
    implementation 'com.google.firebase:firebase-firestore'
```
[ref](https://firebase.google.com/docs/firestore/quickstart)